ADOPTERS.md

```diff
@@ -1,32 +1,5 @@
 # OpenTelemetry Adopters
 
-A non-exhaustive, alphabetized list of organizations that have adopted OpenTelemetry.
+This page has moved, see [Adopters].
```

Comment:
```suggestion
This page has moved [to the opentelemetry.io website][adopters].
```

Reply:
That link text isn't semantically meaningful as a representation of the link target. How about the following:

```suggestion
This page has moved to [opentelemetry.io], see [Adopters].

[adopters]: https://opentelemetry.io/ecosystem/adopters/
[opentelemetry.io]: https://opentelemetry.io
```